Library/ViewModels/BackerDashboardViewModel.swift

```diff
@@ -103,13 +106,16 @@ public final class BackerDashboardViewModel: BackerDashboardViewModelType, Backe
     self.configurePagesDataSource = self.viewDidLoadProperty.signal
       .map { (.backed, DiscoveryParams.Sort.endingSoon) }
 
-    let fetchedUserEvent = self.viewWillAppearProperty.signal.ignoreValues()
-      .switchMap { _ in
-        AppEnvironment.current.apiService.fetchUserSelf()
-          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
-          .prefix(SignalProducer([AppEnvironment.current.currentUser].compact()))
-          .materialize()
-      }
+    let fetchedUserEvent = Signal.merge(
+      self.projectSavedProperty.signal.ignoreValues(),
+      self.viewWillAppearProperty.signal.ignoreValues()
```

Comment:
If we do a `merge`, won't the `fetchUserSelf` network request fire twice for < iOS 13 devices? I wonder if it would be better to use `self.viewDidLoadProperty` so that the network request is made once on initialization, and then again only when a project is saved/unsaved.

Reply:
Actually  `viewDidLoadProperty` won't work in some scenarios, for example:
1-  User opened the open the list of `Saved` projects
2-  Go to Discovery and save another project 
3- Go back to the `Saved` list.

In this case, the list won't refresh (because the notification is posted on the ProjectNavBarViewController).